dll/win32/msi/CMakeLists.txt

```diff
@@ -86,7 +86,7 @@ set_source_files_properties(msi.rc PROPERTIES OBJECT_DEPENDS "${CMAKE_CURRENT_BI
 set_module_type(msi win32dll)
 target_link_libraries(msi uuid ${PSEH_LIB} wine_dll_register oldnames)
 add_dependencies(msi msi_idlheader)
-add_delay_importlibs(msi odbccp32 crypt32 wintrust)
+add_delay_importlibs(msi odbccp32 crypt32 wintrust apphelp)
```

Comment:
Just ftr: WinXP msi does not import anything from apphelp. But since you said, that it is a bit of a hack, that is not necessarily a blocker.

Reply:
It does, but it is done manually.

Reply:
set a breakpoint on `apphelp!ApphelpCheckMsiPackage`, and run a setup.

Reply:
Correct. I missed that you touched the delayed imports here. For a moment I thought you were introducing static linking. My fault.